crates/storage-pg/migrations/20241014145741_upstream_oauth_userinfo.sql

```diff
@@ -0,0 +1,7 @@
+-- Add migration script here
+ALTER TABLE "upstream_oauth_providers"
+  ADD COLUMN "user_profile_method" TEXT NOT NULL DEFAULT 'auto',
+  ADD COLUMN "userinfo_endpoint_override" TEXT;
+
+ALTER TABLE "upstream_oauth_authorization_sessions"
+  ADD COLUMN "userinfo" TEXT;
```

Comment:
```suggestion
  ADD COLUMN "userinfo" JSONB;
```

Reply:
This is going to be annoying : the return of `sqlx::query_as!` then wants a `serde_json::Value`.

But for example `data-model` crate doesn't depend on `serde_json`.

It may be fine to keep it a string ? There is currently not a lot of use of `JSONB` here, and it's not coherent with `id_token`.

Reply:
I'm fine to depend on `serde_json` in `mas-data-model`. I'll be doing something similar in #3521 to store extra query parameters passed on the callback